cli/command.go

```diff
@@ -9,12 +9,12 @@ import (
 
 	"github.com/golang/glog"
 	cli "github.com/jawher/mow.cli"
-	"github.com/kubernetes-incubator/external-storage/lib/controller"
```

Comment:
Should we remove this from `glide.yaml`?

Reply:
I am very sorry, I am not familiar with glide, so basically, I dont know, what and how to do here. :(

Reply:
Ok, we may need to merge this first to a feature branch then and massage that aspect before hitting master.